perf/benchmark/values.yaml

```diff
@@ -42,3 +42,5 @@ client: # client overrides
 
 cert: false
 interceptionMode: REDIRECT
+
+namespace: ""
```

Comment:
set a default value?

Reply:
I don't think we need it. Since we will always set namespace for each test.